Update setup instructions for Braze Cohorts (#4709)
segmentio/segment-docs: src/connections/destinations/catalog/actions-braze-cohorts/index.md

To sync additional Audiences from your Engage space, create a separate mapping in the Braze Cohorts destination. Navigate to **Connections > Destinations**, search and select the Braze Cohorts destination, and follow Steps 9-11 above.

> If you are creating multiple mappings under the same Braze Cohorts destination, then it is recommended to clear the default subscription for all your mapppings. The default subscription is `Event Name is Audience Entered or Event Name is Audience Exited` which gets activated for any event from any Audience connected to your Braze Cohorts destination. To ensure that a mapping is activated only for a specific audience, update the subscription of all your mapings to `Event Property audience_key is` and input the Audience Key copied as instructed in Step 2 above.

Reviewer: ```suggestion
If you are creating multiple mappings in one Braze Cohorts destination, Segment recommends clearing the default subscription for all your mappings from `Event Name is Audience Entered or Event Name is Audience Exited` to `Event Property audience_key is`, replacing `audience_key` with the Audience Key you copied in step 2.
```

Author: Addressed the change. Thank you!
